utils.go

```diff
@@ -154,3 +155,11 @@ func stringifyValue(w io.Writer, val reflect.Value) error {
 	}
 	return nil
 }
+
+func jstr(o interface{}) string {
```

Comment:
Looks like we can remove this helper and the commented-out debug blocks that use it.

Reply:
I think the function should stay, it's really useful for debugging. Also the commented calls in tests are a good reminder where to look at, e.g. for new attributes in the resource.

I'd remove it if we manage to format the json in the HTTP response body.